modules/notification/base/notifier.go

```diff
@@ -23,7 +23,7 @@ type Notifier interface {
 	NotifyRenameRepository(ctx context.Context, doer *user_model.User, repo *repo_model.Repository, oldRepoName string)
 	NotifyTransferRepository(ctx context.Context, doer *user_model.User, repo *repo_model.Repository, oldOwnerName string)
 	NotifyNewIssue(ctx context.Context, issue *issues_model.Issue, mentions []*user_model.User)
-	NotifyIssueChangeStatus(ctx context.Context, doer *user_model.User, issue *issues_model.Issue, actionComment *issues_model.Comment, closeOrReopen bool)
+	NotifyIssueChangeStatus(ctx context.Context, doer *user_model.User, commitID string, issue *issues_model.Issue, actionComment *issues_model.Comment, closeOrReopen bool)
```

Comment:
Hmm, if we sort parameters based on importance, wouldn't it make more sense to add it before or after `closeOrReopen`?

Reply:
My rationale was sort of following the order of operations, user does commit X which then affects issue Y. But I can of course change it to whatever order is preferred.